wings.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
```

```suggestion
buildGo118Module rec {
```

You might need to update nixpkgs pin as well. Got `note: module requires Go 1.18` while building